modules/ROOT/pages/authentication-authorization/manage-users.adoc

```diff
@@ -971,65 +980,66 @@ The `RENAME USER` command is only available when using native authentication and
+<3> label:new[Introduced in 5.24] Removes one, several, or all existing xref:authentication-authorization/auth-providers.adoc[auth provider(s)] from a user.
+However, a user must always have at least one auth provider.
+Therefore, `REMOVE ALL AUTH` must be used in conjunction with at least one `SET AUTH` clause in order to meet this requirement.
+<4> Specifies the password for the user.
+The `'password'` can either be a string value or a string parameter with default value length of at least 8 characters. +
+The `PLAINTEXT` and `ENCRYPTED` keywords are optional and can be used to specify the format of the password, i.e. whether Neo4j needs to hash it or it has already been hashed.
+By default, all passwords are encrypted (hashed) when stored in the Neo4j `system` database.
+* The optional `PLAINTEXT` in `SET PLAINTEXT PASSWORD` has the same behavior as `SET PASSWORD`.
+* The optional `ENCRYPTED` is used to recreate an existing user when the plaintext password is unknown, but the encrypted password is available in the _data/scripts/databasename/restore_metadata.cypher_ file of a database backup.
+See xref:backup-restore/restore-backup#_restore_users_and_roles_metadata[Restore users and roles metadata]. +
+With `ENCRYPTED`, the password string is expected to be in the format of `<encryption-version>,<hash>,<salt>`, where, for example:
+** `0` is the first version and refers to the `SHA-256` cryptographic hash function with iterations `1`.
+** `1` is the second version and refers to the `SHA-256` cryptographic hash function with iterations `1024`.
+<5> Specifies whether the user must change their password at the next login.
+If omitted, the default is `CHANGE REQUIRED`.
+The `SET PASSWORD` prefix of the `CHANGE [NOT] REQUIRED` clause is only optional if it directly follows the `SET PASSWORD 'password'` clause and is not part of a `SET AUTH` clause.
+<6> The default for `SET STATUS` is `ACTIVE`.
+<7> Specifies a home database for a user. A home database is resolved if it is pointing to a database or a database alias. If no home database is set, the DBMS default database is used as the home database for that user.
+
+<8> label:new[Introduced in 5.24] One or more `SET AUTH` clauses can be used to set xref:authentication-authorization/auth-providers.adoc[auth providers], which define authentication / authorization providers for that user.
+This might be used to configure external auth providers like LDAP or OIDC, but can also be used as an alternative way to set the native (password-based) auth settings like `SET PASSWORD` and `SET PASSWORD CHANGE REQUIRED`.
+For examples, see <<access-control-create-users,Creating users>>, xref:authentication-authorization/sso-integration.adoc#auth-sso-auth-providers[Configure SSO at the user level using auth providers], and xref:authentication-authorization/ldap-integration.adoc#auth-ldap-auth-providers[Configure authentication/authorization at the user level using auth providers].
```

Comment:
shouldn't this link down to alter user examples rather than create user examples for the native case?

Reply:
I don't know. This is how it was.

Reply:
might just have been me missing it in previous reviews then 🤷